HISTORY.md

```diff
@@ -1,6 +1,32 @@
 # Release History - `open-autonomy`
 
 
+# 0.6.0 (2022-12-16)
+
+Autonomy:
+- Removes the need for the intermediate agent layer when scaffolding an `FSM` app using specification
```

Comment:
@jmoreira-valory Please create a ticket for the next sprint to update the tutorials accordingly

Reply:
No idea what this update refers to.

Reply:
Currently (well, previously) we needed an intermediate agent to scaffold an FSM App with the scaffold tool.
Apparently this requirement has been eliminated now.

Reply:
A ticket has been created in Trello.

Reply:
Ok then we need to fix the language. "layer" is not a term that's appropriate here